README.md

```diff
@@ -132,9 +126,11 @@ terraform destroy
 ```
 
 As before, you can also type:
+
 ```bash
 terraform destroy -var 'create_appstream=true'
 ```
+
 to skip the create_appstream prompt.
```

Comment:
```suggestion```
there shouldn't be a prompt now `create_appstream` is always defined

Reply:
True, I'll remove this from the README